packages/obonode/obojobo-chunks-math-equation/math-equation-properties-modal.js

```diff
@@ -112,6 +159,16 @@ class MathEquationProperties extends React.Component {
 					</div>
 					<span>{this.state.error}</span>
 				</div>
+				<div id="katexContainer" style={{ display: this.latexCheck(this.state.content.latex) }}>
+					<b id="previewTitle">Preview</b>
+					<div
+						style={{ fontSize: this.state.content.size + 'em' }}
+						aria-hidden
+						id="example-katex"
+						className={'katexPreview'}
+						dangerouslySetInnerHTML={{ __html: this.equationGenerator() }}
```

Comment:
I prefer to use a `get` named method in this case, like `getEquationHTML`, it's more action oriented where as equationGenerator is more passive. I think `generateEquation` would work too but that still doesn't tell you that it is returning HTML, and I like standardizing on using `get` for any function that is getting something for you.

Reply:
https://github.com/ucfopen/Obojobo/wiki/Developer-Code-Style-Guide#function-naming-conventions